template/template_test.go

```diff
@@ -2610,3 +2626,70 @@ arHERAKScrZMTrYPLt2YqYoeyO//aCuT9YW6YdIa9jPQhzjeMKXywXLetE+Ip18G
 eB01bl42Y5WwHl0IrjfbEevzoW0+uhlUlZ6keZHr7bLn/xuRCUkVfj3PRlMl
 -----END CERTIFICATE-----
 `
+
+func TestTemplate_ExecuteSmall(t *testing.T) {
+	t.Parallel()
+
+	now = func() time.Time { return time.Unix(0, 0).UTC() }
+
+	cases := []struct {
+		name string
+		ti   *NewTemplateInput
+		i    *ExecuteInput
+		e    string
+		err  bool
+	}{
+		{
+			"nil",
+			&NewTemplateInput{
+				Contents: `test`,
+			},
+			nil,
+			"test",
+			false,
+		},
+		{
+			"external_func",
+			&NewTemplateInput{
+				Contents: `{{ toUpTest "abCba" }} {{ toLower "LOWER" }}`,
+				ExtFuncMap: map[string]interface{}{
+					"toUpTest": func(inString string) string {
+						return strings.ToUpper(inString)
+					},
+				},
+			},
+			&ExecuteInput{
+				Brain: NewBrain(),
+			},
+			"ABCBA lower",
+			false,
+		},
+	}
+
+	//	struct {
```

Comment:
is this comment still needed? I am assuming it was just forgotten from your testing and can probably be removed